src/snowflake/snowpark/functions.py

```diff
@@ -169,6 +169,7 @@
     <BLANKLINE>
 """
 import functools
+import typing
```

Comment:
just for consistency, we can do `from typing import Literal` as well ?

Reply:
Now, that we cannot. `flake8` does some special checking for type-hints to be pointing at valid things. However; if we did `from typing import Literal` then we bring in `Literal` into a local namespace and it's impossible for tooling to know that it was imported from `typing` originally.

You can see the my bug report at https://github.com/PyCQA/pyflakes/issues/725 and once I dug in deeper into the duplicated issue I must say that I agree with this